services/src/main/java/org/exoplatform/addons/gamification/service/configuration/RealizationsServiceImpl.java

```diff
@@ -19,14 +20,16 @@ public RealizationsServiceImpl(RealizationsStorage realizationsStorage) {
   }
 
   @Override
-  public List<GamificationActionsHistoryDTO> getAllRealizationsByFilter(RealizationsFilter filter,
+  public List<GamificationActionsHistoryDTO> getRealizationsByFilter(RealizationsFilter filter,
+                                                                        Identity identity,
                                                                         int offset,
                                                                         int limit) {
     if (filter == null) {
       throw new IllegalArgumentException("filter is mandatory");
     }
```

Comment:
if (identity == null) {
      throw new IllegalArgumentException("identity is mandatory");
    }

Reply:
done !